libcorn/Cargo.toml

```diff
@@ -24,6 +25,7 @@ wasm-bindgen = { version = "0.2.83", optional = true }
 serde-wasm-bindgen = { version = "0.5.0", optional = true }
 console_error_panic_hook = { version = "0.1.7", optional = true }
 wee_alloc = { version = "0.4.5", optional = true }
+mlua = { version = "0.8.9", features = ["luajit", "vendored", "module", "macros", "serialize"], optional = true }
```

Comment:
Should support for other Lua versions be included (ie at least one non-jit version)? In theory that's as simple as adding more flags here, so I'd say assuming that does just work we should.